assignments/arrays.js

```diff
@@ -63,34 +63,67 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(`Car 33 is a 2011 Jeep Wrangler` );
+
+for (i = 0; i < inventory.length; i++) {
+console.log(`${inventory[i].car_model} was made by ${inventory[i].car_make} in ${inventory[i].car_year}`);
+}
 
 
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = inventory.length -1;
+
+console.log(`${inventory[lastCar].car_model} was made by ${inventory[lastCar].car_make}`);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
-let carModels = [];
-console.log();
+let carModels = [
+];
+
+for (i = 0; i < inventory.length; i++) {
+  console.log(`${inventory[i].car_model}`);
+}
+
 
 // ==== Challenge 4 ====
 // The accounting team needs all the years from every car on the lot. Create a new array from the dealer data containing only the car years and log the result in the console.
 let carYears = [];
-console.log();
+
+for (i = 0; i < inventory.length; i++) {
+  console.log(`${inventory[i].car_year}`);
+}
 
```

Comment:
The correct result will look similar to what you're logging. But the instructions asked for you to return the years in an _array_.

```suggestion
let carYears = [];

 for (i = 0; i < inventory.length; i++) {
  carYears.push(inventory[i].car_year);
};

console.log(carYears);
```